.github/workflows/build-and-upload.yml

```diff
@@ -1,11 +1,11 @@
 name: Build And Upload
 
-#on:
-#  push:
-#    branches:
-#      - main*
-#  workflow_dispatch:
-#  workflow_call:
+on:
+  push:
+    branches:
+      - main*
```

Comment:
This should ideally be only main.
The way it currently is, another main-* branch could overwrite the image published from main and it would be hard to track who's publishing what.